src/app/script/dialog_class.cpp

```diff
@@ -1591,9 +1592,6 @@ int Dialog_set_data(lua_State* L)
 int Dialog_get_bounds(lua_State* L)
 {
   auto dlg = get_obj<Dialog>(L, 1);
-  if (!dlg->window.isVisible())
-    dlg->window.remapWindow();
-
```

Comment:
I've been testing a bit with this bit of code instead:
```c++
  if (!dlg->window.isVisible() && dlg->window.bounds().isEmpty()) {
    dlg->window.remapWindow();
    dlg->window.centerWindow(dlg->parentDisplay());
  }
```
And it seems to fix the case when the script asks for a Dialog's bounds before showing it, avoiding returning empty bounds. If we decide to introduce this change, remember to update the test.